projects/pedal/inc/mech_brake.h

```diff
@@ -0,0 +1,78 @@
+#pragma once 
+
+// Module for the mechanical brake 
+// Detects whether the brake is pressed or unpressed past a set threshold 
+
```

Comment:
This looks like it's copy-pasted from the last car, don't include it in the review if it's not complete. Also, make sure the comments truly reflect what this module does once you ARE done developing it. Imma ignore this file for now. Same goes for `mech_brake.c`